src/module/common/component/input/Button/Button.styles.ts

```diff
@@ -36,7 +36,7 @@ export const ButtonRoot = styled(Button)<ButtonProps>(({ theme, rounded = true }
                 color: "#000000",
             },
             tertiary: {
-                backgroundColor: theme.palette.overlay["12%"],
+                backgroundColor: theme.palette.overlay[100]["12%"],
```

Comment:
To be fixed here: https://github.com/Peersyst/ckbull-mobile/issues/400

Reply:
Done